web/api/v1/api.go

```diff
@@ -762,7 +778,10 @@ func (api *API) serveFlags(r *http.Request) (interface{}, *apiError, func()) {
 
 func (api *API) remoteRead(w http.ResponseWriter, r *http.Request) {
 	api.remoteReadGate.Start(r.Context())
+	remoteReadQueries.WithLabelValues(r.RequestURI).Inc()
```

Comment:
What does a typical `r.RequestURI` look like? Isn't it going to be same every time?

If it's going to have the params or selectors too, then we'll have the infinite-cardinality issue.

Reply:
Yes. you are definitely correct. What would you recommend as a label ? In my previous commit I just added this metric without a label. Let me know if we are better off not having a label for this

Reply:
Done @gouthamve